app/build.gradle

```diff
@@ -45,13 +46,26 @@ dependencies {
     implementation 'com.google.android.material:material:1.3.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
     implementation 'com.google.firebase:firebase-firestore:22.1.1'
+    implementation 'com.google.mlkit:barcode-scanning:16.1.1'
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
     androidTestImplementation 'androidx.test:runner:1.3.0'
     androidTestImplementation 'androidx.test:rules:1.3.0'
     androidTestImplementation 'androidx.test.espresso:espresso-intents:3.3.0'
 
+    // CameraX core library using the camera2 implementation
+    def camerax_version = "1.0.0-rc03"

```

Comment:
Cirrus says "A newer version of androidx.camera:camera-camera2 than 1.0.0-rc03 is available: 1.1.0-alpha02"
True, but I don't know whether we want to be on an alpha build, so it was a conscious decision.
Opinions?